data/p/pros-and-cons-building-vs-buying-your-github-slack-integration-bot.mdx

```diff
@@ -0,0 +1,117 @@
+more than 150k events per day between GitHub events (Pull requests, Pull Requests comments, Pull requests
+code comments, Repository updates, CI/CD, GitHub Actions) and Slack events (messages, edit, and reactions).**
+
+## 2. Send notifications to Slack
+
+The integration would need to send notifications to Slack channels based on the GitHub events being monitored.
+
+{" "}
+
+<ImageContainer
+  alt="slackBlockKitBuilder"
+  src="/blog/static/images/buildvsbuy/slackBlockKitBuilder.jpg"
+  classNameDiv="mx-10 lg:mx-20"
+  classNameImage=""
+  width={3922}
+  height={2128}
+/>
+**Axolo uses the SlackBlockKit Builder to send interactive and user friendly messages, more on [Slack
+Block Kit Builder](https://api.slack.com/tools/block-kit-builder.).**
+
+## 3. Manage user authentication
+
+The integration would need to handle user authentication to ensure that 1. only authorized users can access the integration, 2. developers don't get notified by their own actions from GitHub actions in Slack.
+
+{" "}
+
+<ImageContainer
+  alt="Authentication"
+  src="/blog/static/images/buildvsbuy/Authentication.jpg"
+  classNameDiv="mx-10 lg:mx-20"
+  classNameImage=""
+  width={3464}
+  height={2176}
+/>
+**Axolo authenticates each user onboarded with GitHub(or GitLab) and Slack using oAuth 2.0 mainly to
+avoid getting notified by their own actions from one tool to the other. More about why we do this in
+the [Axolo documentation](https://axolo.co/docs/guides/howto#pros-and-cons-of-githubgitlab-and-slack-signin).**
+
+## 4. Maintain and update the integration
+
+The integration would need to be maintained and updated regularly to keep up with changes to both Slack and GitHub APIs.
+
+{" "}
+
+<ImageContainer
+  alt="changelog"
+  src="/blog/static/images/buildvsbuy/changelog.jpg"
+  classNameDiv="mx-10 lg:mx-20"
+  classNameImage=""
+  width={2816}
+  height={2044}
+/>
+**Learn more about [Axolo regular updates and changelog in our blog](https://axolo.co/blog/tags/changelog).**
+
+# Buying Axolo
+
+Now, let's consider the benefits of buying Axolo instead of building it internally:
+
+## Time-to-market
+
+Building an integration like Axolo from scratch could take months of development time, delaying your ability to use the integration and realize its benefits. By buying Axolo, you can start using the integration immediately, without any additional development effort.
+
+## Reduced development costs
+
+Buying Axolo can save your company significant development costs, as you won't need to hire additional developers to build and maintain the integration. Additionally, Axolo's pricing is typically lower than the cost of hiring additional developers to build and maintain the integration.
+
+## Expert support
+
+By buying Axolo, you gain access to expert support from the Axolo team. This can save your development team time and resources, as they won't need to troubleshoot and fix integration issues on their own.
```

Comment:
```suggestion
By buying Axolo, you gain access to expert support from the Axolo team. This will save your development team time and resources, as they won't need to troubleshoot and fix integration issues on their own.
```